SecuritySystemUWP/SecuritySystemUWP/Storage/OneDrive.cs

```diff
@@ -53,14 +43,13 @@ public DateTime LastUploadTime
             }
         }
 
-        public async void UploadPictures(string camera)
+        public async void UploadPictures(string cameraName)
```

Comment:
This is for a never-implemented feature that isn't on the roadmap. Easier to remove references to it than to add in string checks for something never needed, so I'll remove it.